src/hipscat/pixel_tree/pixel_tree_builder.py

```diff
@@ -81,13 +65,13 @@ def _create_tree(self, partition_info_df: pd.DataFrame):
             partition_info_df: Dataframe loaded from the partition_info metadata
         """
         for _, row in partition_info_df.iterrows():
-            self._create_node_and_parent_if_not_exist(
+            self.create_node_and_parent_if_not_exist(
                 row[PartitionInfo.METADATA_ORDER_COLUMN_NAME],
                 row[PartitionInfo.METADATA_PIXEL_COLUMN_NAME],
                 PixelNodeType.LEAF,
             )
 
-    def _create_node_and_parent_if_not_exist(
+    def create_node_and_parent_if_not_exist(
```

Comment:
What is the reason for changing this to a "public" method?

Reply:
Previously it was used by just the PixelTree class to construct itself, but now that trees will be build from other places it's useful to have.